icu4j/maven-build/maven-icu4j-charset/pom.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0"?>
```

Comment:
Most of the comments in `icu4j\maven-build\maven-icu4j\pom.xml` apply here
Mostly about moving common things in parent.